qt/applications/workbench/workbench/app/mainwindow.py

```diff
@@ -370,6 +370,13 @@ def populate_interfaces_menu(self):
                         action = submenu.addAction(name)
                         action.triggered.connect(lambda checked_cpp, name=name, key=key: self.
                                                  launch_custom_cpp_gui(name, key))
+        # these register scripts contain code to register encoders and decoders to work with project save before the
+        # corresponding interface has been initialised. This is a temporary measure pending harmonisation of cpp/python
+        # interfaces
+        for reg_list in registers_to_run:
+            for register in registers_to_run[reg_list]:
```

Comment:
shouldn't this be?
```suggestion
            for register in reg_list:
```

Reply:
Good spot, thanks

Reply:
I'm a little surprised that even worked like that.

Reply:
Actually I'd misremembered `registers_to_run` as a list rather than a dict, though I've refactored it now to make use of `.values()`